packages/lesswrong/components/sunshineDashboard/ModeratorActions.tsx

```diff
@@ -305,13 +309,13 @@ export const ModeratorActions = ({classes, user, currentUser, refetch, comments,
     <LWTooltip title="Snooze and Approve 1 (Appear in sidebar on next post or comment. User's future posts are autoapproved)" placement="top">
       <SnoozeIcon className={classes.modButton} onClick={() => handleSnooze(1)}/>
     </LWTooltip>
-    {user.needsReview && <LWTooltip
+    <LWTooltip
       title={`${userCommentsWarning ? "Warning: user has made a comment! " : ""}Remove from queue (i.e. snooze without approving posts)`}
     >
       <AlarmOffIcon className={classNames(classes.modButton, {
         [classes.warningButton]: userCommentsWarning,
       })} onClick={handleRemoveNeedsReview}/>
-    </LWTooltip>}
+    </LWTooltip>
```

Comment:
Why make this available for already reviewed users? Seems dangerous - it's not clear from the description that clicking this icon will clear the `reviewedByUserId` for example.

Reply:
I've run into wanting it for newish users who were edge-cases, who I wanted to shift back to content-not-appearing-by-default.

The recent re-styling of this button makes it orange if the user has commented which seemed to be a reasonable amount of warning.

In general I've been finding the moderator-queue trying to have safeguards to be more of a pain than helpful.

Reply:
Can forum-gate this being available for already-reviewed users if it feels like a footgun for the EA forum

Reply:
I think this won't be relevant in the sidebar (or even main ModerationDashboard) because in those two places, user.needsReview is always true (that's how the user shows up there in the first place). The only place where this condition is relevant is in out-of-the-way places where you're either manually investigating a user on the user profile, or in the "recently reviewed" list. I think if you're poking around in that place you're more likely to specifically want this feature (i.e. you accidentally snoozed someone when you meant to remove-from-queue)